student-store-ui/src/components/Product Card/ProductCard.jsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react'
 import "./ProductCard.css"
+import ProductDetail from '../ProductDetail/ProductDetail'
 
-const ProductCard = ({product}) => {
-  return (
-    <div className='product-card'>
-        <h3>{product.name}</h3>
-        <img src={product.image} alt={product.name} />
-    </div>
-  )
+const ProductCard = ({product, shoppingCart, quantity, handleAddItemToCart, handleRemoveItemFromCart}) => {
+    return (
```

Comment:
you can also do `const ProductCard = ({product, shoppingCart, quantity, handleAddItemToCart, handleRemoveItemFromCart}) => (`